app/services/reports/base_report.rb

```diff
@@ -4,6 +4,12 @@ module Reports
   class BaseReport
     private
 
+    def arbitrary_start_day(month:, day:)
+      now = Time.zone.now
+      now.change(year: now.month >= month ? now.year : now.year - 1, month: month, day: day).
+        to_date.to_s
```

Comment:
I'm worried that converting `.to_date` loses some precision and the tests may break around April 1 of next year like [these two did](https://github.com/18F/identity-idp/pull/3685), can we update this to be like `.beginning_of_day` and use a `TimeWithZone` and update the report to take in a time object instead of a string like this?

Reply:
Good ideas!